java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/PartitionSplittingST.java

```diff
@@ -50,13 +52,16 @@ public class PartitionSplittingST {
 
     @BeforeEach
     void setUp(SleeperSystemTest sleeper, AfterTestReports reporting) {
-        sleeper.connectToInstance(MAIN);
+        sleeper.connectToInstanceNoTables(MAIN);
         reporting.reportIfTestFailed(SystemTestReports.SystemTestBuilder::partitionStatus);
     }
 
     @Test
     void shouldSplitPartitionsWith100RecordsAndThresholdOf20(SleeperSystemTest sleeper) {
         // Given
+        sleeper.tables().createWithProperties("test", DEFAULT_SCHEMA, Map.of(
+                TABLE_ONLINE, "false",
+                PARTITION_SPLIT_THRESHOLD, "20"));
```

Comment:
This test has a bit of a problem. We need the table to be offline for the assertion at the end to assert on unreferenced files with the GC schedule enabled, but taking the table offline prevents partition splitting from running. I'm going to merge the partition splitting PR into this one so that splitting can still run with the table offline.